charts/sn-platform/values.yaml

```diff
@@ -1966,6 +1966,8 @@ streamnative_console:
     GOOGLE_REDIRECT_URI: ""
     GRAFANA_AUTH_PROXY: false
     GRAFANA_AUTH_PROXY_USER: ""
+    CONNECTOR_ENABLED: false
```

Comment:
we can set `CONNECTOR_ENABLED` true by default

Reply:
Fixed